app/assets/stylesheets/comments.scss

```diff
@@ -1,4 +1,6 @@
 .comment_stream {
+    border-top: 1px solid #efefef;
+    background: #f9f8f8;
```

Comment:
- Color literals like `#f9f8f8` should only be used in variable declarations; they should be referred to via variable everywhere else.
- Line should be indented 2 spaces, but was indented 4 spaces